### PR TITLE
Touch id commands

### PR DESCRIPTION
**Pull request by dpgraham**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/simulator/touch_id
```
Comment (vmaxim) [inline commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml]: It should be `/session/:session_id/simulator/toggle_touch_id_enrollment`.

File: commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+name: Toggle Touch ID Enrollment
+short_description: Toggle the simulator being [enrolled](https://support.apple.com/en-ca/ht201371) to accept touchId  (iOS Simulator only)
+
+description:
+  |
+    To enable this feature, the `allowTouchIdEnroll` desired capability must be set to true. When `allowTouchIdEnroll` is set to true
+    the Simulator will be enrolled by default, and the 'Toggle Touch ID Enrollment' changes the enrollment state.
+
+    This call will only work if Appium process or its parent application (e.g. Terminal.app or Appium.app) has access to Mac OS accessibility in System Preferences > Security & Privacy > Privacy > Accessibility list
```
Comment (imurchie) [inline commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml]: `if the Appium process`

File: commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  javascript_wd: true
+  javascript_wdio: true
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/simulator/touch_id
+  method: 'POST'
+  json_parameters:
+    - name: match
```
Comment (imurchie) [inline commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml]: Should be an optional `enabled` parameter. https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L427-L429

File: commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml (hunk 2 above)
Comment (imurchie) [inline commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml]: [Pedantic alert] `e.g.` should be followed by a comma (i.e., `(e.g., Terminal.app or Appium.app)`)

**Review by imurchie — CHANGES_REQUESTED** [review]

**Comment by dpgraham** [comment]
Fixes pushed

File: commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3150"
+  ruby: "https://github.com/appium/ruby_lib/blob/master/lib/appium_lib/core/common/command.rb#L64"
+  php: "https://github.com/appium/php-client/" # TODO PHP documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: false
```
Comment (SrinivasanTarget) [inline commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml]: Isn't it false?

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

**Pull request merged**